docs/custom-media.md

```diff
@@ -119,6 +119,8 @@ Each text track entry is a JSON object with the following keys:
     [`text/vtt`](https://developer.mozilla.org/en-US/docs/Web/API/WebVTT_API).
   * `name`: A name for the text track.  This is displayed in the menu for the
     viewer to select a text track.
+  * `default`: Enable track by default.  Optional boolean attribute to enable
```

Comment:
The [validator](https://github.com/calzoneman/sync/blob/3.0/src/custom-media.js#L204) needs to be updated to validate this new field.

However, another thing to consider is that in the proposed change, `default` is an attribute of the text track descriptor itself; nothing inherently prohibits the user from specifying `default: true` on multiple text tracks.  This raises a question of what should happen in that scenario.  One option would be to add validation against it, but that doesn't really prevent the user from making the mistake in the first place.  Another option would be to have a separate top-level `defaultTextTrack` field that points to the URL of one of the `textTracks` entries, which also requires validation to ensure that the default track is actually present in the manifest.